yarn.lock

```diff
@@ -18,6 +16,12 @@
   optionalDependencies:
     chokidar "^2.0.3"
 
+"@babel/code-frame@^7.0.0-beta.35":
```

Comment:
Did you yarn add anything? I only ask because it seems you didn't add anything except newer versions of what existed. webpack 4 is not yet released as stable so we need to be careful bumping up dependencies it relies on. I'd probably check out this file from master (and package.json) and make sure it doesn't install anything unless it's a needed module though.